OpenEXR/IlmImf/ImfMisc.cpp

```diff
@@ -1882,8 +1882,7 @@ getScanlineChunkOffsetTableSize(const Header& header)
     const Box2i &dataWindow = header.dataWindow();
 
     vector<size_t> bytesPerLine;
-    size_t maxBytesPerLine = bytesPerLineTable (header,
-                                                bytesPerLine);
+    (void) bytesPerLineTable (header, bytesPerLine);
```

Comment:
I don't think there's any reason to cast away the result, simply removing the assignment should be enough.

Reply:
done